docs/get_started/install.md

```diff
@@ -424,13 +443,214 @@ $ sudo python setup.py install
+(mxnet)$ pip install mxnet
+```
+
+**Step 5**  Validate the installation by running simple *MXNet* code described [here](#validate-mxnet-installation).
+
+**Note**  You can read more about virtualenv [here](https://virtualenv.pypa.io/en/stable/userguide/).
 
 </div>
 
-<div class="linux">
+<div class="pip">
+<br/>
+
+**Step 1**  Install prerequisites - Homebrew, python development tools.
+
+```bash
+# Install Homebrew
+$ /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+$ export PATH=/usr/local/bin:/usr/local/sbin:$PATH
+
+# Install python development tools - python2.7, pip, python-setuptools
+$ brew install python
+```
+
+**Step 2** Install MXNet with OpenBLAS acceleration.
+
+Installing *MXNet* with pip requires a latest version of `pip`. Install the latest version of `pip` by issuing the following command.
+
+```bash
+$ pip install --upgrade pip
+$ pip install --upgrade setuptools
+```
+
+```bash
+$ pip install mxnet
+```
+
+**Step 3**  Validate the installation by running simple MXNet code described [here](#validate-mxnet-installation).
+
+</div>
+
+<div class="docker">
+<br/>
+
+Docker images with *MXNet* are available at [Docker Hub](https://hub.docker.com/r/mxnet/).
+
+**Step 1**  Install Docker on your machine by following the [docker installation instructions](https://docs.docker.com/docker-for-mac/install/#install-and-run-docker-for-mac).
+
+*Note* - You can install Community Edition (CE) to get started with *MXNet*.
+
+**Step 2** Pull the MXNet docker image.
+
+```bash
+$ docker pull mxnet/python
+```
+
+You can list docker images to see if mxnet/python docker image pull was successful.
+
+```bash
+$ docker images
+
+REPOSITORY          TAG                 IMAGE ID            CREATED             SIZE
+mxnet/python        latest              00d026968b3c        3 weeks ago         1.41 GB
+```
+
+**Step 4** Validate the installation by running simple MXNet code described [here](#validate-mxnet-installation).
+
+</div>
+
+<div class="build-from-source">
+<br/>
+
+Building *MXNet* from source is a 2 step process.
+1. Build the *MXNet* core shared library, `libmxnet.so`, from the C++ sources.
+2. Build the language specific bindings. Example - Python bindings, Scala bindings.
+
+Make sure you have installed Xcode before proceeding further.
+
+<br/>
+
+All the instructions to build *MXNet* core shared library and *MXNet* Python bindings are compiled as one helper *bash* script. You can use [this bash script](https://raw.githubusercontent.com/dmlc/mxnet/master/setup-utils/install-mxnet-osx-python.sh) to build *MXNet* for Python, from source, on macOS.
+
+**Step 1** Download the bash script for building MXNet from source.
+
+```bash
+$ curl -O https://raw.githubusercontent.com/dmlc/mxnet/master/setup-utils/install-mxnet-osx-python.sh
+```
+
+**Step 2** Run the script to get latest MXNet source and build.
+
+```bash
+# Make the script executable
+$ sudo chmod 744 install-mxnet-osx-python.sh
```

Comment:
Removed sudo.

I agree it reduces number of times users need to copy paste. But, I felt this is more clear for user on what is being done.